packages/node/src/cache/flags.ts

```diff
@@ -0,0 +1,32 @@
+import { FlagConfigCache } from 'src/types/cache';
+
+export class DefaultFlagConfigCache implements FlagConfigCache {
+  private cache: Record<string, string> = {};
```

Comment:
document that this cache stores flag configs in stringified form, because the evaluation engine takes flag configs in stringified form.

Reply:
The cache does not cache variants, it caches flag configs. 

Additionally, we don't necessarily need to expose the flag config object to the developer. We get the complete flag config from the server, so all we need to do is  pass the correct flag config objects as JSON to the evaluation engine. 

I changed it to store a `FlagConfig` where `FlagConfig` is a type alias of `Record<string, unknown>` We do need to access some of the object fields from the flag config, e.g. key, and the new flag type (local vs remote) when that gets added.